bakhmut_aleksandr/calculator

user10101011 opened the pull request.

user10101011 committed:
first commit

NikAshanin commented:
Включи у себя сначала swiftlint, потом посмотрю. Пока много мелочи, которую не я должен показывать, а компилятор

user10101011 committed:
Update CalculatorViewController.swift

Устранены предупреждения swiftlint.

user10101011 committed:
Update CalculatorEngine.swift

Устранены предупреждения swiftlint.

user10101011 commented:
Swiftlint подключен, ошибки устранил.

user10101011 committed:
Update GestureViewController.swift

user10101011 committed:
Added OLetterRecognizer.swift, CircleView.swift

OLetterRecognizer.swift содержит код описание жеста.
CircleView.swift содержит код описания круга, отображаемого на экране.

user10101011 committed:
Added Main.storyboard

user10101011 committed:
Added CustomGesture.xcodeproj.zip

user10101011 committed:
Second commit

user10101011 commented:
Последние пять коммитов не нужно смотреть. Не заладились отношения с десктопным гитхабом.)

user10101011 committed:
Upload CircleView.swift

NikAshanin commented:
Ну и что это за жесть в итоге? Зачем было всё ломать?)) Посмотри какие файлы ты закоммитил в этот пулл реквест с калькулятором
